### PR TITLE
Wiki Currency Pages

### PR DESCRIPTION
**Pull request by switchlove**
Adds the missing currency pages to the wiki. 

Added:
- Pokémon Dollars
- Dungeon Tokens
- Quest Points
- Farm Points
- Diamonds
- Battle Points

Replicated all the functionality of the old wiki pages

### CODE REVIEW

**Comment by CypherX** [comment]
Can you move the styling into `styles.css` and the weather code into its own script in `/scripts/`?

I also see on the BP and DT pages there's a section for shopmon but it's only looking at Friseur Furfrou, could these be updated to display all shopmon for the relevant currency?

File: pages/Pokémon Dollars/overview.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                        <td data-bind="text: Math.floor(((Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + 26) + Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + -25))/2)*1.5).toLocaleString(), attr: { 'data-order': $data.routeName }"></td>
+                        <td data-bind="text: Math.floor(((Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + 26) + Math.max(10, 3 * MapHelper.normalizeRoute($data.number, $parents[0]) + 5 * Math.pow(MapHelper.normalizeRoute($data.number, $parents[0]), 1.15) + -25))/2)*2.25).toLocaleString(), attr: { 'data-order': $data.routeName }"></td>
```
Comment (RedSparr0w) [inline pages/Pokémon Dollars/overview.html:75]: The `data-order` in here seems to be out of place, if anything should be `data-sort` (might both be interchangeable).

If it shouldn't be sorted by the text that's already in the column. it should be referencing the data to use in this column
eg `data-sort: PokemonFactory.routeHealth($data.number, $parents[0])` which is just the number without the locale string transformation.

So for the first column this does nothing as the `routeName` is already the text in the column, should probably be used for the number columns though as they will sometimes not sort correctly in deferent locales.

Comment (switchlove) [reply pages/Pokémon Dollars/overview.html:75]: Yeah, that was a test from before I got this branch fully updated. That should be removed by, but still reviewing sorting overall

**Comment by switchlove** [comment]
Okay, should be good?

**Review by RedSparr0w — APPROVED** [review]
Looks great, thanks for all the new pages! 🎉

**Pull request merged**